docs/source/admin/traffic_ops/using.rst

```diff
@@ -881,7 +892,9 @@ The CZF is an input to the Traffic Control CDN, and as such does not get generat
 
 The script that generates the CZF file is not part of Traffic Control, since it is different for each situation.
 
-.. note:: The ``"coordinates"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the closest Cache Group(s) geographically).
+.. note:: The ``"coordinates"`` section is optional and may be used by Traffic Router for localization in the case of a CZF "hit" where the zone name does not map to a Cache Group name in Traffic Ops (i.e. Traffic Router will route to the closest Cache Group(s) geographically).  
+
+The ``"backupList"`` section is optional and is used by Traffic Router for localization in the case of a CZF "hit" when there are no caches available for that DS in the matched cache group.  This backup list is ordered and if there are no caches available in any of the configured backup cache groups, the request will be bypassed (if configured) or rejected
```

Comment:
question: did you build the docs with this change to make sure the `backupList` note is contained within the same box as the `coordinates` note? The syntax highlighting here makes me think that the 2nd note might not be in the same box

Reply:
I built the docs and loaded the file in a browser. Didn't find any issue.